libs/ui-watt/src/lib/components/form-field/form-field.component.scss

```diff
@@ -78,12 +85,27 @@ watt-form-field {
     display: none;
   }
 
+  [wattPrefix],
+  [wattSuffix] {
+    .watt-icon-button {
+      height: 40px;
+    }
+  }
+
   [wattPrefix] {
     margin-right: 8px;
+
+    .watt-icon-button {
+      margin-left: -12px;
```

Comment:
issue: this is applying height to icons inside a `watt-icon-button` component. Often the prefix icon is a `watt-icon` component. Consider that the icon might be inside a `watt-icon-button` component. 

question: Will there be a case where the prefix icon is a `watt-icon-button`?